### PR TITLE
Reviewes API collection response fields

### PR DESCRIPTION
**Pull request by mmadariaga**
Add fields that are present on web admin lists pages but missing in API collection responses

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static function getPropertyMap(string $context = '', string $role = null)
             $response = parent::getPropertyMap(...func_get_args());
         }
 
-        if ($role === 'ROLE_BRAND_ADMIN') {
+        if (in_array($role, ['ROLE_BRAND_ADMIN', 'ROLE_BRAND_ADMIN'])) {
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php:23]: Duplicate ROLE_BRAND_ADMIN in array

File: web/rest/client/config/api/raw/provider.yml [hunk 2]
```diff
@@ -1380,11 +1390,25 @@ Ivoz\Provider\Domain\Model\Queue\Queue:
...
   attributes:
-    access_control: 'FALSE'
-    pagination_enabled: false
-  itemOperations: []
-  collectionOperations: []
+    access_control: >-
+      "ROLE_COMPANY_ADMIN" in roles
+      && user.brandHasFeature('billing')
+      && user.hasAccessPrivileges(_api_resource_class, request.getMethod())
+      && user.getCompany().getShowInvoices()
```
Comment (Kaian) [inline web/rest/client/config/api/raw/provider.yml:1402]: Company.getShowInvoices() is functionally correct to filter RatingPlanGroups?

File: web/rest/client/config/api/raw/provider.yml [hunk 3]
```diff
@@ -1521,6 +1545,27 @@ Ivoz\Provider\Domain\Model\RouteLock\RouteLock:
...
+  itemOperations:
+    get: ~
+  collectionOperations:
+    get: ~
+  attributes:
+    access_control: >-
+      "ROLE_COMPANY_ADMIN" in roles
+      && user.brandHasFeature('billing')
+      && user.hasAccessPrivileges(_api_resource_class, request.getMethod())
+      && user.getCompany().getShowInvoices()
```
Comment (Kaian) [inline web/rest/client/config/api/raw/provider.yml:1558]: Same here for RoutingTags controlled by the showInvoices flag

File: web/rest/client/config/api/raw/provider.yml [hunk 4]
```diff
@@ -1521,6 +1545,27 @@ Ivoz\Provider\Domain\Model\RouteLock\RouteLock:
...
 
+Ivoz\Provider\Domain\Model\RoutingTag\RoutingTag:
+  itemOperations:
+    get: ~
+  collectionOperations:
+    get: ~
+  attributes:
+    access_control: >-
+      "ROLE_COMPANY_ADMIN" in roles
+      && user.brandHasFeature('billing')
```
Comment (Kaian) [inline web/rest/client/config/api/raw/provider.yml:1556]: Not sure if billing feature should filter access to this endpoint

**Review by Kaian — CHANGES_REQUESTED** [review]

File: library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php (hunk 1 above)
Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php]: :+1: the second one was ment to be ROLE_COMPANY_ADMIN

**Review by Kaian — APPROVED** [review]
LGTM :+1:

**Pull request merged**